service/pom.xml

```diff
@@ -13,10 +13,25 @@
     <properties>
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
-        <mapstruct.version>1.5.0.RC1</mapstruct.version>
         <spring-kafka-test.version>2.8.5</spring-kafka-test.version>
         <opencsv.version>5.6</opencsv.version>
+        <spring-kafka.version>2.8.5</spring-kafka.version>
+        <kafka-streams.version>3.1.0</kafka-streams.version>
+        <awaitility.version>4.2.0</awaitility.version>
+        <spring.cloud.version>2021.0.2</spring.cloud.version>
+        <property name="hibernate.enable_lazy_load_no_trans" value="true"/>
```

Comment:
Why is this required?

Reply:
I forgot to remove this. This is not a required line.